java-checks/src/test/files/checks/security/VerifiedServerHostnamesCheck/HostnameVerifier.java

```diff
@@ -0,0 +1,102 @@
+import java.util.function.BiFunction;
```

Comment:
`WARNING HostnameVerifier.java:1: The import java.util.function.BiFunction is never used`

Reply:
good catch, I dropped the line which was using it, for no reason.